Equipment/Item/RandomAffix.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <QString>
+#include <QMap>
+
+#include "ItemStatsEnum.h"
+#include "PhaseRequirer.h"
+
+class Stats;
+
+class RandomAffix : public PhaseRequirer
+{
+public:
+    RandomAffix(unsigned id, Content::Phase required_phase, const QString& name, Stats* stats, const QStringList &stats_list, const QMap<ItemStats, unsigned> &affix_stat_values);
+    unsigned get_id() const;
+    QString get_name() const;
+    QString to_string() const;
+    Stats* get_stats() const;
+    QStringList get_stats_list() const;
+    QMap<ItemStats, unsigned> get_affix_stat_values() const;
+
+private:
+    unsigned id;
+    QString name;
```

Comment:
Since these are never mutated and the getters for them only returns the members without any specific transformation you can make `id` and `name` const members, then move them to public, and then remove the get_id() and get_name() methods. Instead refer directly to the const members where they are used. This reduces the amount of unnecessary getters.

See for example Buff.h which does this for its `name` and `icon` members.